pylibs/plotting/HistogramNormalizer.py

```diff
@@ -44,13 +44,14 @@ def __normalizeToOne(self, hist, sample):
         hist.hist.Scale(1./hist.hist.Integral())
   
   def __normalizeToBackground(self, hist, sample, background_integral):
-    if background_integral is None:
+    if background_integral is None and sample.type != SampleType.background:
```

Comment:
Maybe this method would be a bit clearer if we started with the background case? Something like:

```python
if sample.type == SampleType.background:
    # normalize background
    return


if background_integral is None:
    # print error
    return


# handle non-background cases...
```

Reply:
- I think there should be a return statement at the end of the background case.
- Just a tiny thing, instead of `elif sample.type == SampleType.signal:` we could do: `if sample.type == SampleType.signal:`